test/unit/services/edition_publisher_test.rb

```diff
@@ -63,6 +64,7 @@ class EditionPublisherTest < ActiveSupport::TestCase
     assert edition.published?
     assert_equal '1.1', edition.reload.published_version
     assert_equal 1.week.ago, edition.major_change_published_at
+    assert_equal government, edition.document.government
```

Comment:
This doesn't seem like a great test to add this into. This is publishing a second edition of an already-published document, so in the real world it should have a government already.

I think it would probably be better to make a new test for this.